src/bittorrent_helper.h

```diff
@@ -134,15 +134,21 @@ std::string generatePeerId(const std::string& peerIdPrefix);
 // uses generatePeerId(peerIdPrefix) to produce Peer ID.
 const std::string& generateStaticPeerId(const std::string& peerIdPrefix);
 
+const std::string& generateStaticPeerAgent(const std::string& peerAgent);
+
 // Returns Peer ID statically stored by generateStaticPeerId().  If
 // Peer ID is not stored yet, this function calls
 // generateStaticPeerId("aria2-")
 const unsigned char* getStaticPeerId();
 
+const unsigned char* getStaticPeerAgent();
```

Comment:
Since we use peer agent as std::string, it would be nice to return std::string rather than const unsigned char*